### DISCUSSION TITLE
Missing Script Build

### DISCUSSION THREAD

**Discussion by chris4jahn** [rossjrw/pr-preview-action#92] [Q&A] [answered]
Hi,
I'm pretty new to github actions. 
I use github pages for my blog. The repo can be found here: https://github.com/chris4jahn/Cloud-Coffeebreak
I just added two yml-files in .github/workflows to getting PR preview up and running. I didn't do any other changes. 

When I create a new PR from the "working" branch to "main" I get a workflow error as followed: 

`Run npm i && npm run build
npm warn deprecated debug@[4](https://github.com/chris4jahn/Cloud-Coffeebreak/actions/runs/11105217155/job/30850908475?pr=16#step:3:5).1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm warn deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm warn deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm warn deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1

added 194 packages, and audited 19[5](https://github.com/chris4jahn/Cloud-Coffeebreak/actions/runs/11105217155/job/30850908475?pr=16#step:3:6) packages in 6s

8 packages are looking for funding
  run `npm fund` for details

20 vulnerabilities (1 low, 5 moderate, 13 high, 1 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
npm error Missing script: "build"
npm error
npm error To see a list of scripts, run:
npm error   npm run`

can you help me fixing the issue? Do you need additional information?

Thanks
Chris

**Comment by rossjrw** [comment, marked as answer]
Looks like you are running `npm run build` when you do not have an npm build script defined. I glanced at your repository and you don't appear to be using npm scripts at all. What do you expect `npm run build` to do?
